.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+1. Describe what you expected to happen.
+
+2. Describe what actually happened.
+
+3. Describe what version of Pulp and what OS is affected by the
+   issue.
+
+4. State steps to re-create the issue.
+
+5. If there is a related BZ, or pulp.plan.io add a link to them.
```

Comment:
How about this?

> If there is a related pulp.plan.io issue, link to it, and set the "smash test" field.

The mention of the "smash test" field might be useful. pulp.plan.io issues tend to link to any related BZs, and relying on the pulp.plan.io issue to provide the BZ link means that no information is duplicated.